site/style_about.css

```diff
@@ -0,0 +1,18 @@
+
+body {
+	border:solid 5px;
+	padding-left: 1.5cm;
```

Comment:
While cm and in are valid units of measurement in CSS, they are really only used for print CSS files.  (Not really something we are covering in this course.)  Centimeters and inches are more meaningful on paper; they are less so on a computer screen.  There are really only 3 different units of measurements that we use on screen:

* px (pixels)
* em (relative measurement to the size of the tag you are styling)
* rem (relative size to the size set for the ```<html>``` tag)

We will get to talking about rem and em in an upcoming class.